packages/sdk40/types/query.d.ts

```diff
@@ -0,0 +1,5 @@
+import { google } from "@cosmjs/proto-signing";
+import { Client } from "@cosmjs/tendermint-rpc";
+import { Coin } from "./structs";
+export declare function proveAccount(client: Client, address: string): Promise<google.protobuf.Any>;
+export declare function proveBalance(client: Client, address: string, denom: string): Promise<Coin>;
```

Comment:
"To prove" is the wrong term here, right? If I understand correctly, the chain proves that its answer is correct by providing a proof and the client "verifies" this proof.

Reply:
Good point.

What terms should we use. I want two verbs.

`aaaaAccount`  would signify some way to get the account data that has been validated via merkle and light client proofs (later we just swap in a header-verifying tendermint rpc implementations and we have the full validation here)

`bbbbAccount` would get the account data, just trusting the node it was talking to.

I thought of `proveAccount` and `getAccount`, but you have a valid point that isn't correct terms.

`getAccount` / `getProvenAccount`?

`getAccount` / `verifyAccount` ?

Other ideas?

Reply:
Let's aim high and take verification for granted. Then unverified queries get a warning in the name. E.g.

```
getSequence
getBalance
getUnverifiedBalances
```

Reply:
I will add a non verified command here to demo it (and see if I can translate from the new grpcQuery types).

Reply:
Okay, names changed and grpc query added.

To do this in scale, we are going to need to auto-gen the decorator structs from the cosmos proto files.